src/main/java/dan200/computercraft/core/apis/SoundAPI.java

```diff
@@ -0,0 +1,141 @@
+/**
+ * This file is part of ComputerCraft - http://www.computercraft.info
+ * Copyright Daniel Ratcliffe, 2011-2016. Do not distribute without permission.
+ * Send enquiries to dratcliffe@gmail.com
+ *
+ */
+
+package dan200.computercraft.core.apis;
+
+import dan200.computercraft.ComputerCraft;
+import dan200.computercraft.api.lua.LuaException;
+import dan200.computercraft.api.lua.ILuaContext;
+import dan200.computercraft.shared.computer.core.ServerComputer;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.SoundCategory;
+import net.minecraft.util.SoundEvent;
+
+public class SoundAPI implements ILuaAPI
+{
+
+    /**
+     * Sound API for ComputerCraft. Provides an interface to Forge's API.
+     * Possible use cases are notification apis, alert sounds, or playing music
+     */
+
+    private String[] methods;
+    private ServerComputer computer;
+    private long lastPlayTime;
```

Comment:
Dan generally prefixes fields with `m_`. I don't know how strict he is on this convention, but it might be consistent with the rest of the code.